src/Package/Impl/DataInspect/REnvironmentProvider.cs

```diff
@@ -18,9 +20,7 @@ internal class REnvironmentProvider : IREnvironmentProvider, IDisposable {
             _rSession = session;
             _rSession.Mutated += RSession_Mutated;
 
-            IdleTimeAction.Create(() => {
-                GetEnvironmentAsync().SilenceException<Exception>().DoNotWait();
-            }, 10, typeof(REnvironmentProvider));
+            GetEnvironmentAsync().DoNotWait();
```

Comment:
You do need to silence `MessageTransportException` (and anything else that `GetEnviromentAsync` can be _expected_ to throw). My earlier comment was about catch-all `SilenceException<Exception>`, not about `SilenceException` in general.